extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -64,7 +83,15 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined):
 		return;
 	}
 
-	// TODO: hook up actually adding binding
 	// 5. insert binding
+	try {
+		const result = await azureFunctionsService.addSqlBinding(selectedBinding.type, uri.fsPath, azureFunctionName, objectName, connectionStringSetting);
+
+		if (!result.success) {
+			vscode.window.showErrorMessage(result.errorMessage);
```

Comment:
Return here as well. Same as above though - is returning false ever something that could happen anyways?

Reply:
yeah it returns false if the Azure function couldn't be found or more than one Azure function with that name exists in the file.